docs/contributing.md

```diff
@@ -121,3 +121,75 @@ Prerequisites: dpkg -i http://archive.ubuntu.com/ubuntu/pool/universe/c/checksec
 Always run make sdl-check command before you submit changes.
 
 Prerequisites: pip3 install bandit
+
+## Logging Policy
+
+### Levels
+
+</br>
+
+#### `ERROR`
+Log at this level errors of every ovms operation that failed 
+</br>
+
+Example:
+```
+    "Failed to create a temporary path PATH"
+```
+
+</br>
+
+#### `WARN`
+Log at this level everything that may lead to an error/need to be fix by administrator.
```

Comment:
We could also log warning when we detect using deprecated or partially valid configuration (like when someone passes shape and batch_size). Something that user should be notified about. Not only stuff that may lead to error, but also stuff that can lead to behavior not really wanted or expected by the user.